src/gun_sse_h.erl

```diff
@@ -25,15 +25,19 @@
 }).
 
 %% @todo In the future we want to allow different media types.
-%% @todo For text/event-stream specifically, the parameters must be ignored.
 
 -spec init(pid(), reference(), _, cow_http:headers(), _)
 	-> {ok, #state{}} | disable.
 init(ReplyTo, StreamRef, _, Headers, _) ->
 	case lists:keyfind(<<"content-type">>, 1, Headers) of
-		{_, <<"text/event-stream">>} ->
-			{ok, #state{reply_to=ReplyTo, stream_ref=StreamRef,
-				sse_state=cow_sse:init()}};
+		{_, ContentType} ->
+			case cow_http_hd:parse_content_type(ContentType) of
```

Comment:
I will merge it now but if you want to another test to add would be sending back a content-type that's garbage (for example `content-type: garbage`). I believe it will result in a crash right now. It probably should just return `disable` instead.